index.html

```diff
@@ -23,29 +23,227 @@
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class="portfolio-dashboard">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio-projects.png" alt="" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>
+        <div class="links">
+          <ul class="link-items">
+            <li class="canopy">CANOPY</li>
+            <span class="list-item">•</span>
+            <li class="list-item">Back end dev</li>
+            <span class="list-item">•</span>
+            <li class="list-item">2015</li>
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+    <section class="portfolio-dashboard">
+        <div class="card-header">
+          <div class="header-image">
+            <img src="./Images/Professional-art-printing.png" alt="Dashboard" class="image">
+          </div>
+        </div>
+        <div class="card-body">
+          <div class="title">Multi-Post <br> Stories</div>
+          <div class="links">
+            <ul class="link-items">
+              <li class="canopy">CANOPY</li>
+              <span class="list-item">•</span>
+              <li class="list-item">Back end dev</li>
+              <span class="list-item">•</span>
+              <li class="list-item">2015</li>
+            </ul>
+          </div>
+          <div class="work-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          </div>
+          <div class="courses">
+            <ul class="course-list">
+              <li class="courses-list">html</li>
+              <li class="courses-list">css</li>
+              <li class="courses-list">JavaScript</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+    </section>
+      <section class="about-myself">
+        <div class="card-header">
+          <div class="about-myself-header">
+            <h1>About <br> Myself</h1>

```

Comment:
Please change this to any other `<hx >`tag from `<h2> to <h6>` There should only be one `<h1>` tag in a document, which is the title of your project.  It is important to have best seo for your site as well as to avoid confusing screen readers. Read [here](https://www.quora.com/Why-should-we-only-use-only-1-H1-header-on-a-page) to get more on thsis